src/components/Application/PageElements.tsx

```diff
@@ -64,27 +64,29 @@ const PageElements: React.FC<PageElementProps> = ({
     return (
       <Form>
         <Segment.Group>
-          {elements.map(({ element }) => {
-            return (
-              <Segment key={`question_${element.code}`}>
-                <Grid columns="equal">
-                  <Grid.Column floated="left">
-                    <SummaryViewWrapperNEW {...getSummaryViewProps(element)} />
-                  </Grid.Column>
-                  {element.category === TemplateElementCategory.Question && canEdit && (
-                    <Grid.Column floated="right" textAlign="right">
-                      <Button
-                        content={strings.BUTTON_SUMMARY_EDIT}
-                        size="small"
-                        as={Link}
-                        to={`/application/${serial}/${sectionCode}/Page${pageNumber}`}
-                      />
+          {elements
+            .filter(({ element }) => element.isVisible)
```

Comment:
i think this would be moved to happen before return (see this being needed for reviews also).

not required change

Reply:
Yeah, good point. So put this at the top of the component:
`visibleElements = elements.filter(({ element }) => element.isVisible)`
And replace `elements` with `visibleElements` throughout PageElements?

Reply:
that would be great

Reply:
Done d5c49d4